README.md

```diff
@@ -9,7 +9,15 @@
 This library is still under construction and needs to be peer reviewed as well as have features added.
 
 ## Usage
-This package contains everything you need to get up and running. If you want to report to a system such as Jaeger or Zipkin you will need to use their NuGet packages.
+This package contains everything you need to get up and running. The meta-package includes the following packages:
+* [Jaeger.Core](https://www.nuget.org/packages/Jaeger.Core/)
+* [Jaeger.Senders.Thrift](https://www.nuget.org/packages/Jaeger.Senders.Thrift/)
+
+### Senders
+The implementation of `Jaeger.Core` is agnostic to any reporting endpoint and could be even used without any in case of logfile reporting. For more information about the sender concept, have a look at the sender [README](src/Jaeger.Core/Senders/README.md).
```

Comment:
> src/Jaeger.Core/Senders/README.md

Are you sure these links work without the leading dot?  Maybe GH fixed it.

Reply:
I also tried it yesterday with leading dot if in a sub folder but it’s not basing it correctly. In this case it‘s working since it‘s a relative path to the README and not relative to the document root.

Reply:
Seems like this was introduced 2013: https://github.blog/2013-01-31-relative-links-in-markup-files/